src/graph/ShowExecutor.cpp

```diff
@@ -226,6 +233,214 @@ void ShowExecutor::showEdges() {
     std::move(future).via(runner).thenValue(cb).thenError(error);
 }
 
+
+void ShowExecutor::showCreateSpace() {
+    auto *name = sentence_->getName();
+    auto future = ectx()->getMetaClient()->getSpace(*name);
+    auto *runner = ectx()->rctx()->runner();
+
+    auto cb = [this] (auto &&resp) {
+        if (!resp.ok()) {
+            onError_(Status::Error("Space not found"));
+            return;
+        }
+
+        resp_ = std::make_unique<cpp2::ExecutionResponse>();
+        std::vector<std::string> header{"Space", "Create Space"};
+        resp_->set_column_names(std::move(header));
+
+        std::vector<cpp2::RowValue> rows;
+        std::vector<cpp2::ColumnValue> row;
+        row.resize(2);
+        auto properties = resp.value().get_properties();
+        row[0].set_str(properties.get_space_name());
+
+        std::string buf;
+        buf.reserve(256);
+        buf += folly::stringPrintf("CREATE SPACE %s (", properties.get_space_name().c_str());
+        buf += "partition_num = ";
+        buf += folly::to<std::string>(properties.get_partition_num());
+        buf += ", ";
+        buf += "replica_factor = ";
+        buf += folly::to<std::string>(properties.get_replica_factor());
+        buf += ")";
+
+        row[1].set_str(buf);;
+        rows.emplace_back();
+        rows.back().set_columns(std::move(row));
+        resp_->set_rows(std::move(rows));
+
+        DCHECK(onFinish_);
+        onFinish_();
+    };
+
+    auto error = [this] (auto &&e) {
+        LOG(ERROR) << "Exception caught: " << e.what();
+        DCHECK(onError_);
+        onError_(Status::Error(folly::stringPrintf("Internal error : %s",
```

Comment:
The specific reason for internal error doesn't need to be known by the user

Reply:
An exception has occurred here, we should know what is exception and be easy to handle it.